.github/CODEOWNERS

```diff
@@ -9,3 +9,6 @@
 
 # DevOps for Actions and other workflow changes
 .github/workflows @bitwarden/dept-devops
+
+# BRE Automations
+galaxy.yml
```

Comment:
Does this need DevOps or BRE as a team at the end of the line here?

Reply:
If we put any owner at the end of the line it will block the GitHub Actions bot from merging the PR and require a review manually. We are looking into better solutions, but currently adding the GitHub Actions bot to the `CODEOWNERS` file is not supported.

Reply:
This is how it's currently set up in [server](https://github.com/bitwarden/server/blob/5c9da2e5eaa505a807da2c8897dfdc6c2e5327d3/.github/CODEOWNERS#L60-L62) and our other projects.

Reply:
Makes sense!